Source/SwiftLintBuiltInRules/Rules/Style/IndentationStyleRule.swift

```diff
@@ -0,0 +1,385 @@
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                /*
+                class Foo {
+                \tlet tabLine = true
+                ↓    let spaceLine = true
+                }
+                */
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                class Foo {
+                \tlet tabLine = true
+                \tlet spaceLine = false
+
+                \tfunc barb(arg: String) {
+                \t↓ \tprint(arg)
+                \t}
+                }
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                class Foo {
+                \tlet tabLine = true
+                \tlet spaceLine = false
+
+                \tfunc barb(arg: String) {
+                \t↓ print(arg)
+                \t}
+                }
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                class Foo {
+                \tlet tabLine = true
+                \tlet spaceLine = false
+
+                \tfunc barb(arg: String) {
+                ↓ \t\tprint(arg)
+                \t}
+                }
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                abcd(bar: "far",
+                \t baz: 0,
+                ↓     bat: false)
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                abcd(
+                    bar: "far",
+                ↓\tbaz: 0,
+                ↓\tbat: false)
+                """,
+                testWrappingInComment: false),
+            Example(
+                """
+                // - tab_width: 4
+                abcd(bar: "far",
+                \t↓     baz: 0,
+                \t↓ bat: false)
+                """,
+                configuration: IndentationStyleConfiguration.testTabWidth,
+                testWrappingInComment: false),
+            Example(
+                """
+                // - include_multiline_strings: true
+                class Foo {
+                \tlet multiLineString = \"""
+                \t\tcontent
+                \t\t↓    more content
+                \t\t↓  most content
+                \t\t↓  \ttabbed content
+                \t\t\"""
+                }
+                """,
+                configuration: IndentationStyleConfiguration.testMultilineString,
+                testWrappingInComment: false)
+        ]
+    )
+
+    // MARK: - Initializers
+    // MARK: - Methods: Validation
+    func validate(file: SwiftLintFile) -> [StyleViolation] { // swiftlint:disable:this cyclomatic_complexity
+        var violations: [StyleViolation] = []
+
+        var fileStyle: ConfigurationType.PreferredStyle?
+        if configuration.perFile == false {
```

Comment:
I understand the irony of me arguing any style preferences on this project, but the reason I do this is for cognitive overhead while reading code. I used to always prefer the conciseness of the bang operator for testing conditions, but another programmer shared that they found reading new code much faster when there was a big ol' `false` staring you in the face instead of a tiny little `!`.

I'll go ahead and change it, but I just wanted to share my reasoning :)